packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -115,18 +116,19 @@ export class EthereumAnchorService implements AnchorService {
 
   /**
    * Send requests to an external Ceramic Anchor Service
-   * @param cidStreamPair - mapping
+   * @param params - a RequestAnchorParams object
    * @private
    */
-  private _makeRequest(cidStreamPair: CidAndStream): Observable<AnchorServiceResponse> {
+  private _makeRequest(params: RequestAnchorParams): Observable<AnchorServiceResponse> {
     return defer(() =>
       from(
         fetchJson(this.requestsApiEndpoint, {
           method: 'POST',
           body: {
-            streamId: cidStreamPair.streamId.toString(),
-            docId: cidStreamPair.streamId.toString(),
-            cid: cidStreamPair.cid.toString(),
+            streamId: params.streamID.toString(),
+            docId: params.streamID.toString(),
```

Comment:
FYI, we do not need to have `docId`. It is a legacy that we can ditch right now IMO.